src/io/mod.rs

```diff
@@ -112,7 +112,10 @@ pub(crate) fn build_output(
 }
 
 /// Set up an input stream (microphone) bases on the selected features (cubeb/cpal/none)
-pub(crate) fn build_input(options: AudioContextOptions) -> MediaStream {
+pub(crate) fn build_input(
+    options: AudioContextOptions,
```

Comment:
Not for this PR, but it is a bit weird to have this `AudioContextOptions` struct here

Reply:
I know, but it was just too convenient because we needed all the same fields to construct the input stream. But now we need additional fields so it makes less sense indeed